hudi-common/src/main/java/org/apache/hudi/common/util/ExternalFilePathUtil.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hudi.common.util;
+
+public class ExternalFilePathUtil {
+  private static final String EXTERNAL_FILE_PREFIX = "hudiext_";
+
+  public static String prefixExternalFileWithCommitTime(String fileName, String commitTime) {
```

Comment:
naming: convertExternalFileToHudiFilePath

Reply:
I think that may be misleading since we're not converting this to a traditional hudi file path and this is only a fileName (not full path)